sdk/cognitivelanguage/Azure.AI.Language.Text/tests/TextAnalysisClientLiveTest.cs

```diff
@@ -647,7 +647,7 @@ public async Task AnalyzeText_ExtractiveSummarizationLROTaskAsync()
                     Assert.IsNotNull(extractiveSummarizationLROResult.Results);
                     Assert.IsNotNull(extractiveSummarizationLROResult.Results.Documents);
 
-                    foreach (ExtractedSummaryDocumentResultWithDetectedLanguage extractedSummaryDocument in extractiveSummarizationLROResult.Results.Documents)
+                    foreach (ExtractedSummaryTextResult extractedSummaryDocument in extractiveSummarizationLROResult.Results.Documents)
```

Comment:
Should this be "ExtractedSummaryActionResult" and same for all other skills. 

so the format will be <SkillName>ActionResult

Reply:
fixed